shippable.yml

```diff
@@ -14,16 +14,6 @@ matrix:
         - CI_PULL_REQUEST="$PULL_REQUEST"
         # Set COVERALLS_REPO_TOKEN
         - secure: hfh1Kwl2XYUlJCn4dtKSG0C9yXl5TtksVOY74OeqolvDAdVj4sc+GJD3Bywsp91CJe8YMEnkt9rN0WGI+gPVMcjTmZ9tMUxKiNNBP8m5oLRFbdgKOkNuXjpjpFHHWGAnNhMmh9vjI+ehADo+QIpU1fGxd3yO4tmIJ1qoK3QqvUrOZ1RwUubRXoeVn3xy3LK5yg4vP5ruitbNeWMw/RZZ7D6czvqvEfCgV6b4mdNDRMiqlUJNkaTRc3em1APXr30yagDV3a7hXLq3HdlyFwvF+9pmB4AKhQctyjPN4zvvPd0/gJXq3ZHXSlZXOZBMPXHlSS5pizfSInNszyZyrP3+/w==
-    - go: 1.3
-      env:
-        - KUBE_TEST_API_VERSIONS=v1 KUBE_TEST_ETCD_PREFIXES=kubernetes.io/registry
```

Comment:
the way this currently works, the go1.4 and go1.3 builds run in parallel.

by removing the go1.3 build and putting both prefixes into one build, you'll serialize the two runs, effectively doubling the unit test part of Shippable.